src/displayapp/screens/WatchFacePineTimeStyle.cpp

```diff
@@ -364,13 +364,13 @@ WatchFacePineTimeStyle::~WatchFacePineTimeStyle() {
 }
 
 bool WatchFacePineTimeStyle::OnTouchEvent(Pinetime::Applications::TouchEvents event) {
-  if ((event == Pinetime::Applications::TouchEvents::LongTap) && lv_obj_get_hidden(btnRandom)) {
+  if ((event == Pinetime::Applications::TouchEvents::LongTap) && lv_obj_get_hidden(btnClose)) {
     lv_obj_set_hidden(btnSetColor, false);
     lv_obj_set_hidden(btnSetOpts, false);
     savedTick = lv_tick_get();
     return true;
   }
-  if ((event == Pinetime::Applications::TouchEvents::DoubleTap) && (lv_obj_get_hidden(btnRandom) == false)) {
+  if ((event == Pinetime::Applications::TouchEvents::DoubleTap) && (lv_obj_get_hidden(btnClose) == false)) {
```

Comment:
I think it's more clear and concise to use the `!` operator instead of checking equality to false. Those parentheses are also unnecessary and just make the code look more dense IMHO.

```suggestion
  if (event == Pinetime::Applications::TouchEvents::DoubleTap && !lv_obj_get_hidden(btnClose)) {
```

Reply:
While I agree with the suggestion, and it's also a clang-tidy warning, I don't think it's necessarily in the scope of this PR, so I'll merge without this.